ts/src/bitget.ts

```diff
@@ -2934,13 +3215,14 @@ export default class bitget extends Exchange {
         //         "1399132.341"
         //     ]
         //
+        const volumeIndex = (market['inverse']) ? 6 : 5;
```

Comment:
Is there a reason why you return the volume of the settle currency instead of just the base currency?

@carlosmiei Should it be done like this? ohlcv[5] is the base volume, while ohlcv[6] is the quote currency https://www.bitget.com/api-doc/contract/market/Get-Candle-Data

related to this issue #20556